rustler_bigint/README.md

```diff
@@ -0,0 +1,34 @@
+# Rustler BigInt
+
+`rustler_bigint` provides support for Erlang's arbitrarily-sized integers.
+
+## Installation
+
+Add this to `Cargo.toml`:
+
+```toml
+[dependencies]
+rustler_bigint = { version = "0.1" }
+```
+
+## Example
+
+Lets assume that we need to handle integers of variable size. Some might fit
+into Rust's `i64`, but others might not. For example:
+
+```elixir
+large = 2**65 # This does not fit into i64, it is an Erlang big integer
```

Comment:
maybe use the Bitwise.bsl function because 2**65 is not supported for older elixir versions? 

```
two_to_the_65 =  Bitwise.bsl(2, 65) # This does not fit into i64, it is an Erlang big integer
```